csharp/src/Ice/IObject.cs

```diff
@@ -25,7 +25,6 @@ public interface IRequest
     ///<summary>The base interface for all servants.</summary>
     public interface IObject
     {
-        // See Dispatcher.
```

Comment:
Wasn't this related to the Servant Disp dispatcher

Reply:
Yes, but there is no Servant Dispatcher yet.